src/main/java/com/redhat/cloud/custompolicies/app/rest/utils/PagingUtils.java

```diff
@@ -41,8 +45,8 @@ public static Pager extractPager(@NotNull UriInfo uriInfo) {
         Pager.PagerBuilder pageBuilder = Pager.builder();
         MultivaluedMap<String, String> queryParams = uriInfo.getQueryParameters();
 
-        final String QUERY_PAGE = "page";
-        final String QUERY_PAGE_SIZE = "pageSize";
+        final String QUERY_PAGE = "offset";
```

Comment:
Actually, i don't think the `pageSize` and `offset` are the same, per the [pagination docs](https://platform-docs.cloud.paas.psi.redhat.com/rest/pagination.html#requests) 
> offset indicates the starting position of the query relative to the complete set of items that match the client’s query.

This is actually the number of elements that we are going to "ignore", not the page we want.
You can check their examples to confirm this, see how the first example has `last` and `next` as `offset=10` when the `count=11`.
https://platform-docs.cloud.paas.psi.redhat.com/rest/pagination.html#examples

Reply:
You are right. Good catch